.github/ISSUE_TEMPLATE/bug-repport.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug Report
+about: template to report a bug
+title: "Bug: [BUG NAME]"
+labels: ''
+assignees: ''
+---
+
+### Basic information
+
+* On-premises: Yes/No
```

Comment:
I think the meaning of "On-premises" is not clear to the average person. I'd prefer:

Where did you (try to) install/deploy wire-server? `<!-- please detail operating system version and cloud provider, if any -->`

Also, 
Which installation instructions did you follow? `<!-- e.g. "demo" or "production" instructions from docs.wire.com ? Did you deviate from the instructions at some point? How did you set up TLS certificates and DNS records? Please explain your setup a little. -->`
Which version of kubernetes do you use? `<!-- run 'kubectl version' to find out -->`
Which version of helm do you use? `<!-- run 'helm version' to find out -->`
Did you install kubernetes using ansible+kubespray? `<!-- yes / no. -->`

etc

Reply:
I added most of the point. Regarding *on-premises*, I'd say let's see if people are struggling with it. We can still change it along the way.